src/components/EditCoursePage/EditCoursePage.test.jsx

```diff
@@ -576,6 +576,7 @@ describe('EditCoursePage', () => {
       component.setState({
         submitConfirmVisible: true,
       });
+      component.instance().getData = jest.fn();
```

Comment:
Is there a way to just mock this out once at the top and it holds true for all future calls? I thought we might do something similar for mocking out calls to `now()` or something like that.